sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -23,8 +23,16 @@ class KeyVaultAccessControlClient(KeyVaultClientBase):
     """Manages role-based access to Azure Key Vault.
 
     :param str vault_url: URL of the vault the client will manage. This is also called the vault's "DNS Name".
-    :param credential: an object which can provide an access token for the vault, such as a credential from
+        You should validate that this URL references a valid Key Vault or Managed HSM resource.
+        See https://aka.ms/azsdk/blog/vault-uri for details.
+    :param credential: An object which can provide an access token for the vault, such as a credential from
         :mod:`azure.identity`
+    :type credential: :class:`~azure.core.credentials.TokenCredential`
+
+    :keyword api_version: Version of the service API to use. Defaults to the most recent.
+    :paramtype api_version: ~azure.keyvault.administration.ApiVersion
+    :keyword bool verify_challenge_resource: Whether to verify the authentication challenge resource matches the Key
+        Vault or Managed HSM domain. Defaults to True.
```

Comment:
I thought in the initial proposal there was talk of supplying a list of resources here to verify against - was that determined to be unnecessary?

Reply:
@heaths might know more about the initial proposal -- I've only known about this binary option. I know that Identity is adding a list of supported tenants for cross-tenant auth, but that may be unrelated

Reply:
@annatisch, are you thinking about the Identity SDK change? The Key Vault change is only to make sure the resource (which gets used for the scope) matches the host domain. Separate features.

Reply:
Thanks @heaths - you're totally right, I was confusing this with the Identity change. :)